src/components/Hoverable/index.js

```diff
@@ -98,7 +98,7 @@ class Hoverable extends Component {
                 onMouseEnter={() => this.setIsHovered(true)}
                 onMouseLeave={() => this.setIsHovered(false)}
                 onBlur={(el) => {
-                    if (this.wrapperView.contains(el.relatedTarget)) {
+                    if (!_.isNull(this.wrapperView) && this.wrapperView.contains(el.relatedTarget)) {
```

Comment:
@ahmedGaber93 Should we do this in other `onBlur` too?

https://github.com/Expensify/App/blob/8f9d1c505ace0d545166ad28678f7f4a9cd70d8f/src/components/Hoverable/index.js#L83-L86

Reply:
@mollfpr yes, we should.
I think we need to check null here also to prevent this error in all places
`if (!_.isNull(this.wrapperView) && !this.wrapperView.contains(el.relatedTarget))`
do you agree?

Reply:
@ahmedGaber93 Sounds good to me!

Reply:
@ahmedGaber93 On second thought, looking and the original condition `!this.wrapperView.contains(el.relatedTarget)`, so that's mean null `this.wrapperView` is also `true`?

I'm not sure to understand the condition; it's either
1. `!_.isNull(this.wrapperView) && !this.wrapperView.contains(el.relatedTarget)`
2. `_.isNull(this.wrapperView) || !this.wrapperView.contains(el.relatedTarget)`

What do you guys think?

Reply:
@mollfpr I think the two conditions will prevent call `contains` from null `this.wrapperView`.
But even though we don't have any visual errors, I think the second condition logic is better because it will  `isHovered` to fallback false.

Reply:
@ahmedGaber93 Let's go with the second logic then! Sorry for the hustle 🙏

Reply:
@mollfpr are you mean use `!_.isNull(this.wrapperView) &&` in the two places
or remove it from the second place here.

https://github.com/Expensify/App/blob/8f9d1c505ace0d545166ad28678f7f4a9cd70d8f/src/components/Hoverable/index.js#L83-L86

Reply:
@ahmedGaber93 Only for remove it from this.

```
if (!this.wrapperView.contains(el.relatedTarget)) { 
         this.setIsHovered(false); 
     } 
```

Reply:
Is there any problem if just adding this?
```diff
-                   if (this.wrapperView.contains(el.relatedTarget)) {
+                   if (this.wrapperView && this.wrapperView.contains(el.relatedTarget)) {
```